src/powershell-win-core/NuGet.config

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Do we need this nuget.config? We have one at project root.

Reply:
I think this can be removed, was trying to figure out why it wasn't picking up latest psrp.windows pkg but it turned out I built the pkg wrong.

Reply:
Removed.